var/spack/repos/builtin/packages/netlib-scalapack/package.py

```diff
@@ -58,6 +64,36 @@ def libs(self):
         shared = True if "+shared" in self.spec else False
         return find_libraries("libscalapack", root=self.prefix, shared=shared, recursive=True)
 
+    # patching tests as described in https://github.com/Reference-ScaLAPACK/scalapack/blob/master/README#L147-L157
+    def patch(self):
+        spec = self.spec
+        if "+ilp64" in spec:
+            import os
+
+            file_list = []
+            for root, dirs, files in os.walk(
+                "TESTING/EIG/", topdown=True, onerror=None, followlinks=False
```

Comment:
If you use an abspath here instead of `TESTING/EIG`, then you can replace

```
                    rel_dir = os.path.relpath(root, os.getcwd())
                    rel_file = os.path.join(rel_dir, myfile)
                    file_list.append(rel_file)
```

with 

```
path = os.path.join(root, myfile)
file_list.append(path)
```

which I think is easier to understand